axelor-admin/src/main/java/com/axelor/apps/base/service/app/AnonymizeServiceImpl.java

```diff
@@ -15,10 +19,28 @@
 
 public class AnonymizeServiceImpl implements AnonymizeService {
 
+  FakerService fakerService;
```

Comment:
Is the default access modifier on purpose ? If not please probably need to be protected